problemsets-mat223/linearalbegra.tex

```diff
@@ -2601,24 +2618,96 @@ \section*{Change of Basis \& Coordinates}
 
 		Similarly,
 		\[
-			\mat{\alpha_1\\\vdots\\\alpha_n}_{\mathcal B} = \alpha_1\vec b_1+\cdots +\alpha_n\vec b_n
+			\mat{\alpha_1\\\vdots\\\alpha_n}_{\mathcal B}
+			= \alpha_1\vec b_1+\cdots +\alpha_n\vec b_n
 		\]
-		is notation for the linear combination of $\vec b_1,\ldots,\vec b_n$ with coefficients $\alpha_1,\ldots,\alpha_n$.
+		is notation for the linear combination of $\vec b_1,\ldots,\vec b_n$ with
+		coefficients $\alpha_1,\ldots,\alpha_n$.
 	\end{definition}
 
 	\question
-	Let $\mathcal E=\{\vec e_1,\vec e_2\}$ be the standard basis for $\R^2$ and let 
-	$\mathcal C=\{\vec c_1,\vec c_2\}$ where $\vec c_1=\mat{2\\1}_{\mathcal E}$ and $\vec c_2=\mat{5\\3}_{\mathcal E}$
-	be another basis for $\R^2$.
+	Let $\mathcal E=\Set{\vec e_1,\vec e_2}$ be the standard basis for $\R^2$ and 
+	let $\mathcal C=\Set{\vec c_1,\vec c_2}$ where $\vec c_1=\mat{2\\1}_{\mathcal E}$
+	and $\vec c_2=\mat{5\\3}_{\mathcal E}$ be another basis for $\R^2$.
 	\begin{parts}
 		\item Express $\vec c_1$ and $\vec c_2$ as a linear combination of $\vec e_1$ and $\vec e_2$.
+			\begin{solution}[inline]
+				$\vec c_1=2\vec e_1+\vec e_2$ and $\vec c_2=5\vec e_1+3\vec e_2$.
+			\end{solution}
 		\item Express $\vec e_1$ and $\vec e_2$ as a linear combination of $\vec c_1$ and $\vec c_2$.
+			\begin{solution}[inline]
+				$\vec e_1=3\vec c_1-\vec c_2$ and $\vec e_2=-5\vec c_1+2\vec c_2$.
+			\end{solution}
 		\item Let $\vec v=2\vec e_1+2\vec e_2$. Find $[\vec v]_{\mathcal E}$ and $[\vec v]_{\mathcal C}$.
-		\item Can you find a matrix $X$ so that $X[\vec w]_{\mathcal C} = [\vec w]_{\mathcal E}$ for any
-			$\vec w$?
-		\item Can you find a matrix $Y$ so that $Y[\vec w]_{\mathcal E} = [\vec w]_{\mathcal C}$ for any
-			$\vec w$?
+			\begin{solution}
+				$[\vec v]_{\mathcal E} = \mat{2\\2}$ and 
+				$[\vec v]_{\mathcal C} = \mat{-4\\2}$.
+
+				The second one is since
+				\[
+					\vec v=2\vec e_1+2\vec e_2
+					=2(3\vec c_1-\vec c_2)+2(-5\vec c_1+2\vec c_2)
+					=-4\vec c_1+2\vec c_2.
+				\]
+			\end{solution}
+		\item Can you find a matrix $X$ so that $X[\vec w]_{\mathcal C} = [\vec w]_{\mathcal E}$
+			for any	$\vec w$?
+			\begin{solution}
+				$X = \mat{2&5\\1&3}$ is such a matrix.
+
+				We know $X$ must be a $2\times2$ matrix, so suppose $X=\mat{a&b\\c&b}$
+				for some $a,b,c,d\in\R$. 
+
+				We know from the first part above that
+				\[
+					\mat{1\\0}_{\mathcal C} = \mat{2\\1}_{\mathcal E}
+					\quad \text{and} \quad
+					\mat{0\\1}_{\mathcal C} = \mat{5\\3}_{\mathcal E},
+				\]
+				and that we need $X$ to satisfy
+				\[
+					X\mat{1\\0}_{\mathcal C}=\mat{a\\c}=\mat{2\\1}_{\mathcal E}
```

Comment:
The notation isn't right here. Drop the subscripts.  [...]_C is a true vector and [...]_C=[...]_E. You've forced X to be the identity transformation.